tests/functional/event_handler/test_api_gateway.py

```diff
@@ -681,7 +681,7 @@ def another_one():
 def test_no_content_response():
     # GIVEN a response with no content-type or body
     response = Response(status_code=204, content_type=None, body=None, headers=None)
-    response_builder = ResponseBuilder(response)
+    response_builder = ResponseBuilder(response, serializer=json.dumps)
```

Comment:
can we make serializer optional so our tests remain intact?

Hyrum law... we dunno if any "crazy" person ended up taking dependency on an internal response builder to create their own framework atop.